### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by bizprat**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by L2jLiga — APPROVED** [review]
LGTM

File: docs/Routes.md [hunk 1]
```diff
@@ -34,8 +34,7 @@ fastify.route(options)
 They need to be in
   [JSON Schema](https://json-schema.org/) format, check [here](Validation-and-Serialization.md) for more info.
 
-  * `body`: validates the body of the request if it is a POST or a
-    PUT.
+  * `body`: validates the body of the request if it is a POST, PUT or PATCH method, and doesn't work with GET and HEAD methods.
```
Comment (zekth) [inline docs/Routes.md:37]: Not sure about the formulation. @jsumners as english native could you rephrase this?

**Review by zekth — CHANGES_REQUESTED** [review]

File: docs/Routes.md (hunk 1 above)
Comment (climba03003) [reply docs/Routes.md:37]: maybe @Fdawgs can helps too.

Comment (Fdawgs) [inline docs/Routes.md:37]:
```suggestion
  * `body`: validates the body of the request if it is a POST, PUT, or PATCH method.
```
Second half of the sentence is extraneous, if a HTTP request method is not mentioned in the first half of the sentence then it should be assumed that it is not supported.

File: docs/Validation-and-Serialization.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ The route validation internally relies upon [Ajv](https://www.npmjs.com/package/
 Validating the input is very easy: just add the fields that you need inside the route schema, and you are done!
 
 The supported validations are:
-- `body`: validates the body of the request if it is a POST or a PUT.
+- `body`: validates the body of the request if it is a POST, PUT or PATCH method, and doesn't work with GET and HEAD methods.
```
Comment (Fdawgs) [inline docs/Validation-and-Serialization.md:127]:
```suggestion
- `body`: validates the body of the request if it is a POST, PUT, or PATCH method.
```
Second half of the sentence is extraneous, if a HTTP request method is not mentioned in the first half of the sentence then it should be assumed that it is not supported.

**Review by Fdawgs — CHANGES_REQUESTED** [review]

**Review by l198881 — APPROVED** [review]

**Comment by l198881** [comment]
Done.

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Fdawgs — APPROVED** [review]

**Review by jsumners — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
@zekth you good now?

**Review by zekth — APPROVED** [review]
LGTM. Sorry for long time review
